components/attention/w-attention.vue

```diff
@@ -37,15 +38,10 @@ const directionName = computed(() => directions.find((e) => props[e]))
 
 const attentionClasses = computed(() => ({
   [props.attentionClass]: true,
-  [ccAttention.notCallout]: !props.callout,
+  [ccAttention.notCallout]: !props.callout
 }))
 
-const wrapperClasses = computed(() => ({
-  [ccAttention.base]: true,
-  [ccAttention.tooltip]: props.tooltip,
-  [ccAttention.callout]: props.callout,
-  [ccAttention.popover]: props.popover,
-}))
+const wrapperClasses = computed(() => [ccAttention.base, [getVariantClasses(props).wrapper]])
```

Comment:
Redundant square brackets? 🤔 (Sorry for picking on details. I'm done now. Promise. 😅🙈)
```suggestion
const wrapperClasses = computed(() => [
  ccAttention.base,
  getVariantClasses(props).wrapper
]);
```

Reply:
Ah, those brackets tricked me after all! 😄 It's great that you spot these things - I should've taken another look before pinging you! Fixed in [7a7f49c](https://github.com/warp-ds/vue/pull/96/commits/7a7f49c0630c2a444022be48e3074b670f51dac9)